asset/api/Asset.Samples/AnalyzeIamPolicyLongrunningGcs.cs

```diff
@@ -56,8 +53,14 @@ public class AnalyzeIamPolicyLongrunningGcsSample
         var operation = client.AnalyzeIamPolicyLongrunning(request);
         // Wait for it to complete
         operation = operation.PollUntilCompleted();
-        // Return the metadata(request)
-        return operation.Metadata;
+
+        // Check whether the operation failed.
```

Comment:
Ditto. (If this is something we do in a lot of samples, let's talk about that.)

Reply:
Some samples do this, but to explicitly write some output, which is better than writing that output on a catch I think. But here, it makes no sense because we are simply throwing.